lib/filter-headers.js

```diff
@@ -5,7 +5,7 @@ module.exports = function filterHeaders (attributes, headers) {
     if (attributes.public) {
         return newHeaders;
     };
-    ['accept-language', 'referer', 'user-agent'].forEach((key) => {
+    ['accept-language', 'accept-encoding', 'referer', 'user-agent'].forEach((key) => {
```

Comment:
We shouldn't proxy the browser, because we don't care for the fragments if the browser supports gzip.